lib/utilities/wrappers.js

```diff
@@ -0,0 +1,96 @@
+import assert from 'bsert';
+
+// TODO: bitcoincash import doesn't work?
+const MTX = {
+  bitcoin: require('bcoin/lib/primitives/mtx'),
```

Comment:
Interesting that bcash import doesn't work. I do wonder if doing the full path could cause issues with not looking for browser versions where applicable though. Webpack will fill that in automatically which is useful. Another thought is that while it's not necessary here, using imports instead of would allow for async imports.